formbuilder/app.js

```diff
@@ -7,6 +7,7 @@ const createHashHistory = require("history/lib/createHashHistory");
 import routes from "./routes";
 import configureStore from "./store/configureStore";
 import "./styles.css";
+import "./bootstrap.css";
```

Comment:
I am not sure to understand.

The purpose here is to import in app.js to let webpack add the file into the bundle. But I was not sure about this.